src/index.html

```diff
@@ -2,16 +2,161 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
+    <script type="text/javascript" src="./main.js"></script>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname">
+        </label>
+
+        <label class="form-field">
+           Name:
+          <input type="text" name="name">
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+            placeholder="mm / dd / yyyy"
+           >
```

Comment:
```suggestion
          <input
            type="date"
            name="date"
            placeholder="mm / dd / yyyy"
          >
```
здається тут в тебе не на тому ж рівні, хоча не впевнений на 100